README.md

```diff
@@ -16,6 +16,12 @@ For an example of how it is used, checkout [brickman].
 This is currently in the development stages and is unstable. The device driver
 interfaces require the latest ev3dev kernel (currently v3.16.7-ckt4-ev3dev1).
 
+## Dependencies
+
+Besides dependencies included in upstream, we need a more recent vala, which you
```

Comment:
What is "upstream"? Ubuntu trusty? I think it would be better to say "trusty" here if that is the case.

Can we put the minimum vala version here too (0.24)?